src/main/webapp/WEB-INF/views/manufacturer/allManufacturers.jsp

```diff
@@ -0,0 +1,31 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All Manufacturers</title>
```

Comment:
you have separeted folder for manufacturers
follow one codestyle in app
change name for jsps:
- from allManufacturers to all 
- from createManufacturer to create